config/locales/en.yml

```diff
@@ -645,16 +645,21 @@ en:
       email_body: |
         Hello %{preferred_name},
 
-        Thanks for starting your taxes with GetYourRefund - you’re almost there! We will prepare your taxes once we have all of your required information and tax documents.
+        Thanks for starting your taxes with GetYourRefund - you’re almost there! Your case number is %{client_id}. We will prepare your taxes once we have all of your required information and tax documents.
 
-        If you have more documents to send or have any questions about your case you can email your tax preparer at hello@getyourrefund.org, chat with our specialists using the chat box on getyourrefund.org/tax-questions or submit more documents at your secure submission link: %{requested_docs_link}
+        You can securely upload your ID and additional tax documents here: %{requested_docs_link}
+
+        If you have any questions you can contact your tax team via email <a href="mailto:hello@getyourrefund.org>hello@getyourrefund.org</a> or text message 58750.
 
         Remember, we’ll review everything with you before filing so just do the best you can. Then we’ll help you get the tax credits that belong to you!
 
         We’re here to help!
-        Your tax team at GetYourRefund.org
+        Your Tax Team at GetYourRefund.org
```

Comment:
I'm surprised we capitalize Your Tax Team like that here, when we don't capitalize it like that elsewhere in https://github.com/codeforamerica/vita-min/blob/3cf253221cffba978e9e2848c3bdffdd7fdd1679/config/locales/en.yml

Not a blocker, but wanted to raise it and see if you're sure!

Reply:
Confirmed with Anu - we will keep it capitalized.